pages/osx/xcrun.md

```diff
@@ -0,0 +1,23 @@
+# xcrun
+
+> Run or locate development tools and properties.
+
+- Get help:
+
+`xcrun --help`
+
+- Get version:
+
+`xcrun --version`
+
+- Show verbose output:
+
+`xcrun --verbose`
+
+- Find tool for given SDK:
```

Comment:
What kind of tool, or isn't this specified?

Reply:
This isn't specified ("find the tool for the given SDK name" is what `xcrun --help` and `man xcrun` output)

Reply:
```suggestion
- Find a tool for a given SDK:
```